src/app/components/ExerciseCard/ExerciseCard.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import 'tailwindcss/tailwind.css';
+
+type ExerciseCardProps = {
+  thumbnail: string;
+  name: string;
+  category: string;
+};
+
+function ExerciseCard({
+  thumbnail,
+  name,
+  category,
+}: ExerciseCardProps): JSX.Element {
+  return (
+    <article className="w-32 h-44 rounded-xl grid">
+      <div className="">
+        <img
+          className="w-44 h-24 rounded-t-xl object-cover"
+          src={thumbnail}
+          alt=""
+        />
+      </div>
+      <div className="pl-1 text-sm font-bold">{name}</div>
```

Comment:
You could add a headline here
```suggestion
      <h3 className="pl-1 text-sm font-bold">{name}</h3>
```